IPython/core/ultratb.py

```diff
@@ -152,23 +150,18 @@ def _format_traceback_lines(lines, Colors, lvals, _line_format):
             res.append('%s   (...)%s\n' % (Colors.linenoEm, Colors.Normal))
             continue
 
-        line = stack_line.text.rstrip('\n') + '\n'
-
-        new_line, err = _line_format(line, 'str')
-        if not err:
-            line = new_line
-
+        line = stack_line.render(pygmented=True).rstrip('\n') + '\n'
         lineno = stack_line.lineno
         if stack_line.is_current:
             # This is the line with the error
             pad = numbers_width - len(str(lineno))
             num = '%s%s' % (debugger.make_arrow(pad), str(lineno))
-            line = '%s%s%s %s%s' % (Colors.linenoEm, num,
-                                    Colors.line, line, Colors.Normal)
```

Comment:
`Colors.line` (typically red) was meant to highlight the current line, but it was completely overridden by PyColorize highlighting every token. With pygments the syntax highlighting overrides `Colors.line` inconsistently, so we remove it. No one should notice a difference unless there's a way to have colors excluding syntax highlighting.